app/javascript/controllers/projects/samples/metadata/create_controller.js

```diff
@@ -0,0 +1,38 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["metadataToAdd", "fieldsContainer", "fieldTemplate"]
+
+  connect() {
+    this.addField()
```

Comment:
**nitpick:** When a new field set is added can you autofocus on the key input please.

Reply:
Added in 29a8aed5011098652a3ad377856a8f9bcdcffd8a

Reply:
Awesome, thanks.